scripts/next-with-deps.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+START_DIR=$PWD
+# gets last argument which should be the project dir
+for PROJECT_DIR in $@;do :;done
+
+if [ -z $PROJECT_DIR ];then
+  echo "No project directory provided, exiting..."
+  exit 0;
```

Comment:
Should probably use exit 1 for error messages

Reply:
Updated to use `exit 1` for these